client/components/main/style.scss

```diff
@@ -18,6 +18,14 @@
 	margin-top: 0;
 }
 
+@font-face {
```

Comment:
This is kinda unrelated to this PR, but ensures the Recoleta font face is defined on all pages (we were previously setting in on My Home).